File: problem_a.cpp
/* Problem A
 *
 * Check if a sentence is valid (i.e. contains no repeated words).
 * If so, return "yes", otherwise return "no".
 */


#include <iostream>
#include <sstream>
#include <string>
#include <vector>


using namespace std;


bool isValid(string sentence)
{
    istringstream iss(sentence);
    vector<string> words;
    copy(istream_iterator<string> (iss),
         istream_iterator<string> (),
         back_inserter(words));

    int i = 0;

    while (i < words.size())
    {
        if (count(words.begin(), words.end(), words[i]) > 1)     // Repeated word found
            return false;

        i++;
    }

    return true;    // No repeated words
}


int main()
{
    string sentence;
    getline(cin, sentence);

    cout << (isValid(sentence)? "yes" : "no") << endl;
}

File: problem_a.py
# Problem A

# Check if a sentence is valid (i.e. contains no repeated words).
# If so, return "yes", otherwise return "no"


import sys


def isvalid(sentence):
    """
        Returns False if repeated words are found and True otherwise.
    """

    for word in sentence:
        if sentence.count(word) > 1:
            return False
    return True


if __name__ == '__main__':
    fptr = sys.stdout

    sentence = input().split()

    if isvalid(sentence):
        fptr.write('yes' + '\n')
    else:
        fptr.write('no' + '\n')

    fptr.close()